scripts/jenkins/groovy/pipelineUtils.groovy

```diff
@@ -127,20 +127,6 @@ class PipelineUtils {
         return gradleVersion
     }
 
-    boolean dockerImageExistsInRegistry(final context, final String registry, final String projectName, final String repositoryName, final String version) {
```

Comment:
Do we need to delete this?

Reply:
I saw its deleted [here](https://github.com/h2oai/h2o-3/pull/16714/changes#diff-606c84166941026c89de1b31075a970ccfe062158c0c77efa300b1be74470c68L46). I guess its ok